core/domain/question_services.py

```diff
@@ -217,3 +226,141 @@ def update_question(
     updated_question = apply_change_list(question_id, change_list)
     _save_question(
         committer_id, updated_question, change_list, commit_message)
+
+
+def get_new_question_id():
```

Comment:
Where is this used?

Reply:
I've used this in creator_dashboard.py in NewQuestionHandler that creates a new question.